public/locales/fr/translation.json

```diff
@@ -84,34 +87,40 @@
         },
         "session": {
             "auth": "Authentification",
-            "expire": "Votre session va expirer, voulez-vous la renouveler ?",
+            "expire": "Votre session va expirer. Voulez-vous la renouveler ?",
             "update": "Renouveler la session"
         }
     },
     "home": {
         "app_subtitle": "Collez un mot de passe, un message confidentiel ou des données privées.",
-        "welcome": "Garantir que vos données sensibles restent cryptées, sécurisées et confidentielles.",
-        "bummer": "C'est dommage !",
-        "maintxtarea": "Inscrivez vos informations sensibles..",
-        "lifetime": "Temps de vie",
+        "welcome": "Assurez-vous que vos données sensibles restent chiffrées, sécurisées et confidentielles.",
+        "bummer": "C'est dommage !",
+        "maintxtarea": "Écrivez vos informations sensibles..",
```

Comment:
The double period looks a bit strange here. But it is the same in English. Maybe a typo in the English file that was blindly replicated by the AI you used? I left it untouched but I'm happy to fix it in all language files if you confirm this is unwanted.

Reply:
Yes, it is most likely directly translated by AI. I will remove the .. in English as well. While looking into the English text it is a bit awkward, so feel free to use the correct terms in French here.

Reply:
Sure. I can fix the double period typo in English as well if you want. I should be able to take care of that today.

Reply:
I did some updates, so if you update your branch you will get the latest English changes